clang/test/SemaSYCL/spurious-host-warning.cpp

```diff
@@ -56,5 +59,39 @@ void foo()
 // expected-warning@+2 {{'simple_dual_port' attribute ignored}}
 #endif
   [[intel::simple_dual_port]] unsigned int v_ten[64];
-}
 
+#ifndef SYCLHOST
+// expected-warning@+2 {{'bank_bits' attribute ignored}}
+#endif
```

Comment:
Rather than do this, it's usually better to use a custom verify. e.g, `-verify=host` on the RUN line and then `// host-warning {{'bank_bits' attribute ignored}}` instead of using the preprocessor conditional.

Reply:
I see we do this pattern all over the file, so feel free to do this cleanup as an NFC commit later if you'd like.

Reply:
Thanks @AaronBallman for the reviews. I will follow-up this as NFC commit later